Sources/Target/POSIX/ELFProcess.cpp

```diff
@@ -53,43 +53,56 @@ inline ErrorCode GetELFSharedLibraryInfoAddress(ELFProcess *process,
   PHDR ph;
   DYNSYM dynsym;
 
-  CHK(process->readMemory(process->loadBase(), &eh, sizeof(eh)));
-
-  if (eh.e_phnum == 0) {
+  // 1. Find the PHDR, number of entries, size of
```

Comment:
For your comments, you can use the full 80 columns. It seems like you set your editor to something like 65 columns or so.